features/localization/managing_countries/editing_country.feature

```diff
@@ -0,0 +1,26 @@
+@addressing
+Feature: Editing country
+    In order to enable or disable countries
+    As an Administrator
+    I want to be able to edit a country
+
+    Background:
+        Given I am logged in as administrator
+
+    @ui
+    Scenario: Disabling country
+        Given the store has "France" country enabled
```

Comment:
I'm still voting for using previously defined step in geographical context to create a country, and the just use step `And the "France` is disabled/enabled`